DOCS-2993: add templates for API operations (#28)
MultiSafepay/openapi: docs/object-schema-template.md

> # This is a template for defining a component with schema type: object, that is OpenAPI compliant.

Reviewer: ```suggestion
# This is an OpenAPI template for a schema type: object component.
```

# To use this file:
#  1. Make a copy of it.
#  2. Delete these comments.
#  3. Rename the file extension to *.yaml.
description: "Add a description."
type: object
properties:
  example_parameter:
    description: "Add a description."

>     example: Add an example
>     type: Choose a type

Reviewer: ```suggestion
    type: Specify the type.
    example: Add an example.
```

required:

>   - List which parameters are required

Reviewer: ```suggestion
  - List which parameters are required.
```
